pkgs/native_assets_cli/test/model/build_output_test.dart

```diff
@@ -159,4 +159,15 @@ version: ${BuildOutput.version}'''),
       throwsFormatException,
     );
   });
+
+  test('BuildOutput dependencies can be modified', () {
+    // TODO(https://github.com/dart-lang/native/issues/168):
```

Comment:
Probably leave a link to https://github.com/dart-lang/native/issues/25 instead.

Reply:
Good idea, thanks!